frontend/src/app/core/model/meetup.ts

```diff
@@ -0,0 +1,138 @@
+import {Location} from './location';
+import {ClymbingGym} from './clymbing-gym';
+import {User} from './user';
+
+/**
+ * Encapsulate a single meetup
+ */
+export class Meetup {
+  /**
+   * The unique id of the meetup
+   */
+  private _id: string;
+  /**
+   * The owner of the meetup
+   */
+  private _owner: User;
+  /**
+   * The date/time when the meetup starts
+   */
+  private _from: Date;
+  /**
+   * The date/time when the meetup ends
+   */
+  private _to: Date;
+  /**
+   * The geo location where the meetup happens, if it is an outdoor event
+   */
+  private _outdoor: Location;
+  /**
+   * The climbing gym where the meetup happens, if it is an indor event
+   */
+  private _indoor: ClymbingGym;
```

Comment:
Hier würde ich einen String oder Number als Type wählen. Im Meetup sollten wir nur einen Code für die Halle ablegen. Die Hallen legen wir in einer separaten Map (fix im Code oder auf der DB) als key value ab, wobei der key als code ins Meetup kommt und der value den Hallennamen darstellt z.b. 'die Kletterhalle (St. Gallen)'. Mit dem Code lässt sich nachher einfacher nach Meetups suchen, die Halle wird ein Hauptkriterium sein in der Suche.

Reply:
Habe hier den Typ String für _indoor verwendet. Habe die nicht mehr benötigten Klassen gelöscht. Die Hallen Map sollten wir anschließend in der DB speichern.